src/common/ortho.jl

```diff
@@ -1,2 +1,3 @@
 # Orthonormalize
-ortho_qr(φk) = Matrix(qr(φk).Q)
+ortho_qr(φk::AbstractArray) = Matrix(qr(φk).Q) #LinearAlgebra.QRCompactWYQ -> Matrix
```

Comment:
Couldn't we simply do this?
```
ortho_qr(φk::Array) = Matrix(qr(φk).Q) 
ortho_qr(φk::T) where T <: AbstractGPUArray = T(qr(φk).Q) 
```
Another way to do it would be to have only one function and to get the the "base type" of `φk`, then convert `qr(φk).Q` to this type: this can be done by calling T.name.wrapper (or maybe one day a dedicated function in Base). We would then have the following code:
`ortho_qr(φk::T) where T <: AbstractArray = T.name.wrapper(qr(φk).Q)`